userguide/tutorials/PostgreSQL.adoc

```diff
@@ -0,0 +1,66 @@
+==== PostgreSQL Configuration
+
+In order to configure PostgreSQL, you need to follow the steps given below (These steps can be executed either via the command line *psql* tool or via *pgAdmin*):
+
+. Create a user corresponding to Kill Bill. You can run the following command :
+[source,sql]
+ CREATE ROLE killbilluser WITH LOGIN INHERIT CREATEDB CREATEROLE NOREPLICATION PASSWORD 'killbill';
+
+ . Create a database corresponding to Kill Bill. You can run the following command:
+[source,sql]
+ CREATE DATABASE killbill WITH OWNER = killbilluser;
+
+
+. Connect to the Kill Bill database created above. In psql, you can run the following:
+[source,sql]
+  \connect killbill;
+
+. Create the Kill Bill schema. Run the following command:
+[source,sql]
+  CREATE SCHEMA killbillschema authorization killbilluser;
+
+. Connect to the schema created above using the following command:
+[source,sql]
+  set schema 'killbillschema';
+
+. Execute https://github.com/killbill/killbill/blob/master/util/src/main/resources/org/killbill/billing/util/ddl-postgresql.sql[this] schema extension DDL.
+
+. Run the Kill Bill DDL https://docs.killbill.io/latest/ddl.sql[here]. In psql, you  can run the following command:
+[source,sql]
+  \i <ddl_file_path>
+
+
+. Grant privileges to the schema using the following command:
+[source,sql]
+  GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA killbillschema TO killbilluser;
+
+==== PostgreSQL kaui Configuration
+
+In order to configure PostgreSQL, you need to follow the steps given below (These steps can be executed either via the command line *psql* tool or via *pgAdmin*):
+
+.  Create a database corresponding to kaui. You can run the following command:
+[source,sql]
+ CREATE DATABASE kaui WITH OWNER = killbilluser;
+
+.  Connect to the kaui database created above. In psql, you can run the following:
+[source,sql]
+  \connect kaui;
+
+.  Execute https://github.com/killbill/killbill/blob/master/util/src/main/resources/org/killbill/billing/util/ddl-postgresql.sql[this] schema extension DDL.
+
+.  Run the Kaui DDL https://github.com/killbill/killbill-admin-ui/blob/master/db/ddl.sql[here].
+
+.  Grant privileges to the schema using the following command:
+[source,sql]
+  GRANT ALL ON SCHEMA public TO killbilluser;
+
+.  Modify the `PROJECT_ROOT/profiles/killbill/src/main/resources/killbill-server.properties` OR `TOMCAT_HOME\conf\catalina.properties` as the case may be. Add the following properties :
+[source,properties]
+ #KAUI postgresql section
+  kaui.db.url=jdbc:postgresql://127.0.0.1:5432/kaui
+  kaui.db.username=killbilluser
+  kaui.db.password=killbill
+
```

Comment:
Could you add a similar section to the Kill Bill PostgreSQL configuration section above just for the sake of consistency?